src/Entities/Pawn.cpp

```diff
@@ -468,9 +468,10 @@ void cPawn::HandleFalling(void)
 		m_LastGroundHeight = GetPosY();
 
 		// Farmland trampling. Mobs smaller than 0.512 cubic blocks won't trample (Java Edition's behavior)
+		// We only have width and height, so we have to calculate Width^2
 		if (GetWorld()->IsFarmlandTramplingEnabled() &&
 			(BlockAtFoot == E_BLOCK_FARMLAND) &&
-			(GetWidth() * GetHeight() >= 0.512))
+			(std::pow(GetWidth(), 2) * GetHeight() >= 0.512))
```

Comment:
pow is slower than just repeated multiplication, it's only necessary when you need to do variable-exponent powers or noninteger exponents.

Reply:
Well, that's fine. The performance would probably end up the same after compiler's optimizations though. Fixed

Reply:
Yep, probably not too much different in reality. :+1: